extensions/sql-migration/src/dialog/createSqlMigrationService/createSqlMigrationServiceDialog.ts

```diff
@@ -214,6 +214,7 @@ export class CreateSqlMigrationServiceDialog {
 
 		const subscriptionDropdownLabel = this._view.modelBuilder.text().withProps({
 			value: constants.SUBSCRIPTION,
+			description: (IconPathHelper.info, constants.MIGRATION_SERVICE_SUBSCRIPTION_INFO),
```

Comment:
should this and all other description: string | undefined values be
description: constants.MIGRATION_SERVICE_SUBSCRIPTION_INFO
the IconPathHelper.info should not be needed.

Reply:
Done. Thanks